src/EditorFeatures/Test/RenameTracking/RenameTrackingTestState.cs

```diff
@@ -176,7 +176,9 @@ public async Task AssertNoTag()
         public async Task<IList<Diagnostic>> GetDocumentDiagnosticsAsync(Document document = null)
         {
             document = document ?? this.Workspace.CurrentSolution.GetDocument(_hostDocument.Id);
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
same here?

Reply:
Yeah all pragmas should now be reverted given RenameTracking analyzer is now a document diagnostic analyzer.

Reply:
Thanks, I forgot to remove these lines when I removed `[Obsolete]` from `RenameTrackingDiagnosticAnalyzer`.